authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/GarminSourceAuthorizationService.kt

```diff
@@ -53,6 +65,13 @@ class GarminSourceAuthorizationService(
         }
     }
 
+    fun checkForUsersWithElapsedEndDateAndDeregister() {
+        requestScope.runInScope(Runnable {
+            val users = userRepository.queryAllWithElapsedEndDate(GARMIN_AUTH)
+            users.forEach { user -> deRegisterUser(user) }
```

Comment:
A more kotlin-like syntax would be
```suggestion
            userRepository
                .queryAllWithElapsedEndDate(GARMIN_AUTH)
                .forEach { deRegisterUser(it) }
```